sdk/.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
   "plugins": ["@azure/azure-sdk"],
-  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"]
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "ignorePatterns": ["**/test/perf/track-1"]
```

Comment:
@HarshaNalluru, should we also add track 2 here?

Reply:
No, track-2 gets compiled just like the regular tests, we don't want them to be ignored.

Reply:
And btw, with this https://github.com/Azure/azure-sdk-for-js/pull/13770 proposal, we'll manage the perf tests differently.
So, it doesn't even matter.

Reply:
Alright, then can you update #13770 to revert the change in this PR?